requirements/dev-requirements.txt

```diff
@@ -2,7 +2,7 @@ twine
 
 pylint==2.13.5
 pydocstyle 
-flake8 
+flake8==4.0.1
```

Comment:
hmm? Must we have that exact version?

Reply:
Bug in new version, I do what JKL tells me